### PR TITLE
#22 module task - test included

### PR DESCRIPTION
**Pull request by nored91**

### CODE REVIEW

File: src/event/event.dto.ts [hunk 1]
```diff
@@ -19,21 +17,18 @@ export class CreateEventDTO {
 
   @IsString()
   @Length(1, 250)
```
Comment (KevinLegras) [inline src/event/event.dto.ts:19]: Add @IsDefined or @IsOptional

File: src/task/test/task.e2e.spec.ts [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+  let app: INestApplication;
+  let eventRepository: Repository<Event>;
+  let taskRepository: Repository<Task>;
+  let userRepository: Repository<User>;
+  let groupRepository: Repository<Group>;
+  let eventForTask: Event = null;
+  let userForTask: User = null;
+  let groupForTask: Group = null;
+
+  let groupPartialForTask: Partial<Event> = {
```
Comment (KevinLegras) [inline src/task/test/task.e2e.spec.ts:27]: it's not an event it's a group

**Review by KevinLegras — COMMENTED** [review]
update the comment pls

File: src/event/event.dto.ts (hunk 1 above)
Comment (nored91) [reply src/event/event.dto.ts:19]: done

File: src/task/test/task.e2e.spec.ts (hunk 2 above)
Comment (nored91) [reply src/task/test/task.e2e.spec.ts]: done

**Review by KevinLegras — APPROVED** [review]

**Pull request merged**